backend/mora/service/orgunit.py

```diff
@@ -837,13 +837,12 @@ def terminate_org_unit(unitid):
       }
 
     """
-    date = util.get_valid_from(flask.request.get_json())
+    date = util.get_valid_from(flask.request.get_json(), is_end=True)
 
-    c = lora.Connector(virkningfra=util.to_iso_time(date),
-                       virkningtil='infinity')
+    c = lora.Connector(effective_date=util.to_iso_date(date))
```

Comment:
Actually, we could just do `date + util.ONE_DAY` here, and do away with the `is_end` parameter above, altogether - and similarly for terminating employees.

Reply:
My point was, having a parameter called `is_end` for a function used to get a _from_ date, is slightly confusing in my opinion - it's not clear to me how to actually interpret this semantically, other than the parameter just being "return the date as usual, but with one day added"
The only places we actually call `get_valid_from` with this parameter is in these termination methods, where we still do a bunch of date arithmetic anyway.
So, to me this is an opportunity to get rid of a parameter to a function, keeping it simpler.

Reply:
Well, as an alternative we could make add a parameter to the function for specifying the key — which is `from` rather than the customary `to` in this case. Would that make more sense?

Reply:
For future reference: we changed the API instead.